src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -113,7 +113,7 @@
 helps['keyvault create'] = """
 type: command
 short-summary: Create a key vault.
-long-summary: Default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
+long-summary: If `--enable-rbac-authorization` is not specified, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
```

Comment:
Long summary is not shown in CLI reference. May you can move it to short summary.

Reply:
> Long summary is not shown in CLI reference. May you can move it to short summary.

@qwordy Good point, but I think it's ok to keep it as long summary, for the docs display issue, we'd better push docs team to display our long summaries as well. At lease we can see the long summary using `-h`.

Reply:
![image](https://user-images.githubusercontent.com/3250203/79948330-3c571900-84a6-11ea-9e08-4ae12fa999ed.png)

Reply:
OK. No problem.